freshrss/umbrel-app.yml

```diff
@@ -0,0 +1,73 @@
+manifestVersion: 1
+id: freshrss
+category: networking
```

Comment:
there's also the `social` and `files` (files and productivity) categories that we could consider

Reply:
I went with `social` because it would be getting feeds so kind of `social media` application